Jenkinsfile

```diff
@@ -1,7 +1,6 @@
 // Builds a module using https://github.com/jenkins-infra/pipeline-library
 buildPlugin(useContainerAgent: true, configurations: [
-    [platform: 'linux', jdk: 11], 
```

Comment:
We're not dropping JDK11 until there is an LTS release that is no longer supporting Java11.